docs/Helix.md

```diff
@@ -120,3 +120,16 @@ There are two main ways to opt out of helix
 - Skipping an individual test via `[SkipOnHelix("url to github issue")]`.
 
 Make sure to file an issue for any skipped tests and include that in a comment next to either of these
+
+## Process for updating helix matrix
+
+Goal is to balance cost/flakiness against having some coverage of supported distros:
+- At the start of each product version, we pick a set of queues/versions/arches to run based on popularity and perceived risk, and how long is left in the support for that OS version.
+- Whenever a new OS is coming online, we ask CTI to do a run on it, and we submit a PR to add a queue for it to check for any failures in it, but if there aren’t any, we don’t merge it.
```

Comment:
- While I understand the need for a Helix queue for non-CTI testing, this reads like nobody else would create one for the new OS version. Suggest we call that out as optional.
    - No need to add a Helix queue if we _only_ use CTI tests to confirm the new OS version is fine.
    - When we need a Helix queue, it would be because we plan to at least open a dotnet/aspnetcore PR to try the queue out. Wouldn't the Helix queue update (from us or whoever) need to be merged for a PR in our repo❔
    - More generally, isn't it CTI testing "and / or" try out a dotnet/aspnetcore PR using a Helix queue for the new OS version❔

Reply:
Sure did you have a particular wording you had in mind?  The gist of it sounds like we will look at updating the queues at the start of the product cycle, and mostly use CTI to validate new asks during each cycle.  I think the main value of this blurb is mostly to capture intent, as opposed to calling out the specifics of how we are going to do each of these steps, since we should just 'do the right thing'

Reply:
But yeah for #2, there's no possibility of exercising a new os distro without the helix docker image in place, which if we are being proactive and doing this at the start of release cycles, will probably be us that needs to spin them up...